package.json

```diff
@@ -49,9 +53,11 @@
     "babel-loader": "^9.2.1",
     "canvas": "^2.8.0",
     "htmlparser2": "^4.1.0",
+    "sinon": "^19.0.2",
```

Comment:
Do we need this yet?

Reply:
I installed it because it was included in the Tap file, but I will remove it if I don't end up using it.